astropy/time/formats.py

```diff
@@ -15,7 +15,7 @@
 
 from astropy.utils.decorators import lazyproperty, classproperty
 from astropy.utils.exceptions import AstropyDeprecationWarning
-from astropy import units as u
+import astropy.units as u
```

Comment:
Still a bit surprised about this: I've been changing it in the other direction in my own code...